### PR TITLE
Save tweets of users and tweets of particular hashtag in separate folder closes #29

### LINKED ISSUE #29: Save tweets of users and tweets of particular hashtag in separate folder.
After getting tweets of user or getting tweets of particular hashtag, a csv file is created for each. These files are directly saved in the root of the project directory. There must be a separate directory for these files and they must be ignored by github.

### PR DESCRIPTION
**Pull request by Griffintaur**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Dependig on the system(Windows or Linux) change the backward or forward slash appropriately.
+Tweets: /TweetsStore/
```
Comment (0x006E) [inline config.yml:2]: Windows can use both back and forward slashes I think. So there won't be any change required

File: twweet_cli.py [hunk 2]
```diff
@@ -56,7 +60,16 @@ def get_all_tweets(screen_name):
     outtweets = [[tweet.id_str,tweet.created_at,tweet.text.encode("utf-8")]for tweet in alltweets]
 
     # write to csv
-    fn=str(screen_name+'_tweets.csv')
+    global TweetsStorage
+    TweetsStorage=str(os.getcwd()+TweetsStorage)
+    os.makedirs(os.path.dirname(TweetsStorage), exist_ok=True)
+#    if not os.path.exists(os.path.dirname(TweetsStorage)):
+#        try:
+#            os.makedirs(os.path.dirname(TweetsStorage))
```
Comment (0x006E) [inline twweet_cli.py]: Remove unnecessary comments please.

**Review by 0x006E — APPROVED** [review]
The code is great and I think it wud work crrctly. But do remove those unnecessary comments. :+1:

File: twweet_cli.py (hunk 2 above)
Comment (Griffintaur) [reply twweet_cli.py]: These codes work for Python 2.7
Are there multiple branches for this project having Python 2.7. if yes then we can port these commented codes to that branch
The commented codes will work for Python 2.7

Comment (0x006E) [reply twweet_cli.py]: We removed the branch for python 2.7. Now the master branch is python 3.x

**Comment by Griffintaur** [comment]
I have removed the unwanted comments

**Comment by 0x006E** [comment]
Join the gitter chat If any new suggestions or any doubt about the program :+1:

**Comment by 0x006E** [comment]
Why 99 and 66?? I jzt didn't undrstnd the logic of that.

**Comment by Griffintaur** [comment]
I will add that as part of the other commit
@nithinswarrier  Nothing special, we can use any number we like

**Comment by 0x006E** [comment]
Mention CruiseDevice. He will merge your commit after checking it

**Comment by Griffintaur** [comment]
@CruiseDevice can u have a look onto the commit

**Comment by 0x006E** [comment]
Ankit, please also update the README.MD. Add the dependency for the new changes. It won't be automatically installed.

**Comment by Griffintaur** [comment]
@nithinswarrier  i will update the Readme.
 Dependency list in the setup file is already modified and is part of commit

**Comment by CruiseDevice** [comment]
@Griffintaur merged. :+1:  :sparkles:

**Pull request merged**